internal/npm_install/generate_build_file.ts

```diff
@@ -60,6 +60,11 @@ const BAZEL_VERSION = args[6];
 const PUBLIC_VISIBILITY = '//visibility:public';
 const LIMITED_VISIBILITY = `@${WORKSPACE}//:__subpackages__`;
 
+const isModuleRegExp = /^export|^import(?!declare globa)/m;
+const declaresGlobalRegExp = /^declare global/m;
+const isNotOnlyDeclaredModuleRegExp = /^\w(?!eclare module)/m;
```

Comment:
intentional missing `d`?

Reply:
Intentional.  It is consumed by the `\w`, that is this matches lines that start with a word-character but aren't immediately followed by `eclare module`